_inc/lib/tracks/class.tracks-client.php

```diff
@@ -184,6 +184,7 @@ static function get_connected_user_tracks_identity() {
 		}
 
 		return array(
+			'blogid' => Jetpack_Options::get_option( 'id', 0 ),
```

Comment:
Nit: now that we're here, should we align those `=>` vertically according to our coding standards?